src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/NewAzureIntegrationAccountCommand.cs

```diff
@@ -51,6 +51,7 @@ public class NewAzureIntegrationAccountCommand : LogicAppBaseCmdlet
 
         [Parameter(Mandatory = false, HelpMessage = "The integration account SKU.",
             ValueFromPipelineByPropertyName = true)]
+        [ValidateSet("Free", "Basic", "Standard", IgnoreCase = true)]
```

Comment:
If a user provides a string outside these three values will the cmdlet always error? If it previously would not always error, this is a breaking change.  However, if the cmdlet would ALWAYS error when there was a SKU provided outside these three string values, this is fine.

Reply:
It would always error even without this ValidateSet, this is just documenting the existing limitations. I would assume that adding in the ValdiateSet would change the error message returned. Before the API would return an error stating that the sku isn't allowed and now the PS cmdlet will. Just wanted to confirm that is alright?

Reply:
Yup! As long as you aren't erroring in cases where no error was previously thrown before, it is fine for a different error to be thrown (and it is nice not to have to make the call to the server and that the valid values are documented).